docs/snippets/book/getting-started/Converter.Tests/TemperatureConverterTests.cs

```diff
@@ -0,0 +1,61 @@
+namespace Converter.Tests;
+
+public class TemperatureConverterTests
+{
+    #region ConvertCToF_Zero_Returns32
+    [Test]
+    public void ConvertCToF_Zero_Returns32()
+    {
+        // Arrange
+        var sut = new TemperatureConverter();
+
+        // Act
+        var result = sut.ConvertCToF(0);
+
+        // Assert
+        Assert.That(result, Is.EqualTo(32));
+    }
+    #endregion
+    #region ConvertCToF_100_Returns212
+    [Test]
+    public void ConvertCToF_100_Returns212()
+    {
+        // Arrange
+        var sut = new TemperatureConverter();
+
+        // Act
+        var result = sut.ConvertCToF(100);
+
+        // Assert
+        Assert.That(result, Is.EqualTo(212));
+    }
+    #endregion
+    #region RemainingExamples
+    [Test]
+    public void ConvertCToF_37_Returns98point6()
+    {
+        // Arrange
+        var sut = new TemperatureConverter();
+
+        // Act
+        var result = sut.ConvertCToF(37);
+
+        // Assert
+        Assert.That(result, Is.EqualTo(98.6));
```

Comment:
Since this is a decimal, the result may or may not match.  A good time to use the Within
```cs
Assert.That(result, Is.EqualTo(98.6).Within(0.1);
```

Reply:
Yes, technically. I think that gets into docs territory or another example scenario though. As is the test passes and AFAIK will always pass. Too many concepts to introduce for a "first use". 

Though I do think it speaks to the need for a "common pitfalls and remedies" article that maybe briefly delves into summarizing some of the analyzer checks and additional things to watch for.

Reply:
Sure, when doubles are used in particular one should be aware that the EqualTo may not match, decimals I think are better, but not quite sure.  It all depends.  Perhaps have a pointer to a Tricks and Traps section for that?

Reply:
And just for fun:  
![image](https://user-images.githubusercontent.com/203432/224512950-0b18c56e-746a-4f8b-9683-e3ef95844f0e.png)